lib/public/404.html

```diff
@@ -0,0 +1,11 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <h1>Not found page</h1>
+</body>
+</html>
```

Comment:
EOF error. in posix systems, file is expected to be ended with new line symbol.
Make that as a setting in your editor.

Reply:
fixed